### PR TITLE
test: Cypress - Unskip & Fix Flaky Tests

### PR DESCRIPTION
**Pull request by Vijetha-Kaja**
## Description

**Fixed below flaky tests**
- PromisesApp_spec.js
- Command_Click_Navigation_spec.js
- AutoFillWidgets_Reflow_spec.ts
- Replay_Editor_spec.js
- ForkTemplateToGitConnectedApp.js
 
## Type of change

- Flaky test fix

## How Has This Been Tested?
- Cypress test runs

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/cypress/e2e/Regression/Apps/PromisesApp_spec.js [hunk 1]
```diff
@@ -89,41 +90,37 @@ describe("JSEditor tests", function () {
...
-      "GREEN",
-    ); */
+    cy.wait(1000);
+    // verify text is visible
+    agHelper.AssertContains(
+      "Step 4: Value is Green and will default to GREEN",
+      "be.visible",
+      ".t--draggable-textwidget span",
+    );
+    cy.get(commonlocators.tableNextPage).click();
```
Comment (sharanya-appsmith) [inline app/client/cypress/e2e/Regression/Apps/PromisesApp_spec.js:111]: can agHelper be used here and sleep instead of cy.wait? 

Rest all looks good.

**Review by sharanya-appsmith — APPROVED** [review]

File: app/client/cypress/e2e/Regression/Apps/PromisesApp_spec.js (hunk 1 above)
Comment (Vijetha-Kaja) [reply app/client/cypress/e2e/Regression/Apps/PromisesApp_spec.js:111]: Sure, i'll make the change in my other open [PR](https://github.com/appsmithorg/appsmith/pull/26541)

**Pull request merged**